sites/public/lib/appAutofill.ts

```diff
@@ -3,13 +3,14 @@ import {
   Application,
   ApplicationStatus,
   ApplicationSubmissionType,
-  Language,
 } from "@bloom-housing/backend-core/types"
 import { blankApplication } from "@bloom-housing/ui-components"
 
 class AutofillCleaner {
   application: Application = null
 
+  // provide context value to override current application language choosen by user on the first step
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I don't see anything untyped here…do we still need the disable comment?

Reply:
nice catch! Removing it...